infrastructure/zk/src/run/run.ts

```diff
@@ -94,10 +93,13 @@ export async function tokenInfo(address: string) {
 }
 
 // installs all dependencies and builds our js packages
-export async function yarn() {
+export async function yarn(sdk: boolean) {
```

Comment:
I'd specify true as default value, so we don't need to change all the previous usages

Reply:
updated